bugbug/repository.py

```diff
@@ -1011,6 +1012,48 @@ def clone(repo_dir):
     clean(repo_dir)
 
 
+def apply_stack(repo_dir, stack, branch):
```

Comment:
We have to analyze all patches even if one was already available in the repo, so we should do something like:
- find parent of the first patch in the list returned by json-automationrelevance (the info is available in the data returned by json-automationrelevance);
- update mercurial repo to ^;
- apply the stack of patches.

Reply:
> update mercurial repo to ^;

Or "tip" if that's not available (but we should assert that it is when the repo is autoland).

Reply:
Ok, so that's even simpler than what i thought (like in libmozevent)